source/data/webdav_access_to_irods.rst

```diff
@@ -50,7 +50,7 @@ How to access iRODS using WebDAV on a Windows 10 pc:
 .. image:: webdav/map1.png
 
 - Choose the drive name you want to use. 
```

Comment:
You have to mention here that because the password that are created are temporal they need to not select the option connect at sign-in and another option that save the password. Otherwise the first password get saved and there is no way to change (at least not that I know). They need to ask a new password (trought the web) everytime they want to reconnect the drive

Reply:
reprocessing the same steps is required once the password is expired. Accordingly updated.